scripts/start-auto-claude.sh

```
@@ -0,0 +1,89 @@
+  cd auto-claude
+  python3 -m venv .venv >> "$LOG_FILE" 2>&1
+  source .venv/bin/activate
+  pip install -q -r requirements.txt >> "$LOG_FILE" 2>&1
+  cd ..
+  log "${GREEN}✓ Python backend ready${NC}"
+fi
+
+# Step 4: Install UI dependencies (if needed)
+if [ ! -d "auto-claude-ui/node_modules" ]; then
+  log "${YELLOW}Installing UI dependencies (this may take a minute)...${NC}"
+  cd "$PROJECT_DIR/auto-claude-ui"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)